ver/us/undefined_syms.txt

```diff
@@ -62,21 +62,6 @@ D_09002020 = 0x09002020;
 D_09004040 = 0x09004040;
 D_09008BE0 = 0x09008BE0;
 
```

Comment:
can more syms be removed from here, given you did more gfx? or maybe there aren't any added here for those effects yet?

Reply:
Yep, this was the only effect with undefined syms in this PR. We just have a few more (around 4) to clean.